src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -8192,6 +8192,31 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     // IF_SVE_BI_2A
     // MOVPRFX <Zd>, <Zn>
     theEmitter->emitIns_R_R(INS_sve_movprfx, EA_SCALABLE, REG_V3, REG_V5);
+
+    // IF_SVE_BH_3A
+    // ADR <Zd>.<T>, [<Zn>.<T>, <Zm>.<T>{, <mod><amount>}]
+    theEmitter->emitIns_R_R_R_I(INS_sve_adr, EA_SCALABLE, REG_V0, REG_V2, REG_V3, 0, INS_OPTS_SCALABLE_S);
```

Comment:
Happy as is, but it'd be nice to have some high registers here to make sure nothing is overflowing.

Reply:
I modified one to test V31 and capstone outputted: `40ACFF04  adr   z0.d, [z2.d, z31.d, lsl #3]` so it all should be good.